pkg/blockbuilder/tsdb.go

```diff
@@ -52,6 +52,8 @@ var softErrProcessor = mimir_storage.NewSoftAppendErrorProcessor(
 	func([]mimirpb.LabelAdapter) {}, func([]mimirpb.LabelAdapter) {}, func(error, int64, []mimirpb.LabelAdapter) {},
 	func(error, int64, []mimirpb.LabelAdapter) {}, func(error, int64, []mimirpb.LabelAdapter) {},
 	func(error, int64, []mimirpb.LabelAdapter) {}, func(error, int64, []mimirpb.LabelAdapter) {},
+	func(error, int64, []mimirpb.LabelAdapter) {}, func(error, int64, []mimirpb.LabelAdapter) {},
```

Comment:
Question to reviewers: Is this having any performance impact? We're passing more pointers on the heap I assume?